pkg/controller/certificatemanager/certificatemanager_test.go

```diff
@@ -52,7 +52,7 @@ var _ = Describe("Test CertificateManagement suite", func() {
 		appSecretName  = "my-app-tls"
 		appSecretName2 = "my-app-tls-2"
 		appNs          = "my-app"
-		csrImage       = "quay.io/tigera/key-cert-provisioner:master"
+		csrImageRegex  = `quay\.io/tigera/key-cert-provisioner:(master|release-.*)`
```

Comment:
this may not fully work because the versions can also be v1.1.1 for instance on released images. Perhaps there are tests that test for this already and the image tag is available?

Reply:
I can tweak the regex to also allow e.g. v1.1.1, or to just ignore the tag altogether, but I'm not sure what the test really needs to do.

Any Operator devs know? Maybe @rene-dekker , @caseydavenport , @tmjd ?

Reply:
That was indeed silly of me to include in the test.. Could you replace it with:
```
			imageSet, err := imageset.GetImageSet(context.Background(), cli, installation.Variant)
			Expect(err).NotTo(HaveOccurred())
			expectedImage, err := components.GetReference(
				components.ComponentCSRInitContainer,
				installation.Registry,
				installation.ImagePath,
				installation.ImagePrefix,
				imageSet,
			)
			Expect(initContainer.Image).To(Equal(expectedImage))
```

And remove line 57:
```
		csrImage       = "quay.io/tigera/key-cert-provisioner:master"
```

Reply:
Thanks @rene-dekker !